Open-Rival/include/Image.h

```diff
@@ -4,6 +4,7 @@
 #include <fstream>
 #include <string>
 #include <vector>
+#include <memory>
```

Comment:
Can we keep these alphabetised?

The way I've been organising the includes is:

```
pch
"This" header file (for the current .cpp file)

Library / system headers

Open-Rival headers
```